nservicebus/wormhole/index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Wormhole Gateway
+summary: How to send messages between geographically separate sites using Worm Hole Gateway 
+component: Wormhole
+reviewed: 2017-05-20
+---
+
+`NServiceBus.Wormhole` is an alternative to the [Gateway](/nservicebus/gateway/) component of NServiceBus. Following table compares the most important aspects of both technologies.
```

Comment:
> component of NServiceBus. 

component of NServicebus, *which does what if you don't know what the Gateway is?*

> Following table compares

*The* following table compares